test/metabase/app_db/test_util.clj

```diff
@@ -74,12 +74,15 @@
 (defn liquibase-file->included-ids
   "Read a liquibase migration file and returns all the migration id that is applied to `db-type`.
   Ids are orderer in the order it's defined in migration file."
-  [file-path db-type]
-  (let [content (u.yaml/from-file (io/resource file-path))]
+  [file-path db-type conn]
+  (let [content (u.yaml/from-file (io/resource file-path))
+        lb-type (if (= "MariaDB" (.getDatabaseProductName (.getMetaData ^java.sql.Connection conn)))
+                  :mariadb
+                  db-type)]
```

Comment:
Does `db-type` take the value `:mysql` now when using MariaDb?

How did Mariadb migrations work before?

Reply:
Yeah, we seem to use `:mysql` for `db-type` for MariaDB. 

Previously, liquibase would count the changeset with just a `sql dbms="mariadb"` as executed when ran on all databases, and only on mariadb would it actually execute anything. With the changed logic, it only marks it ran if it's ran on mariadb. The hoisted dbms attribute to the changeset level also has that same behavior, but more consistently since the "what do we do when there is only non-ran changesets?" logic is what is buggy in liquibase.

So, now our tests needed to recognize mysq vs mariadb for tracking what will be marked as ran or not